app/models/audit_trail_event.rb

```diff
@@ -0,0 +1,6 @@
+class AuditTrailEvent < ApplicationRecord
+  belongs_to :auditable, polymorphic: true
+  belongs_to :user
+
+  serialize :event_changes, Hash
```

Comment:
Would like to change this to changes. Because I'm repeating the `event` from the Model name.